scripts/helpers/clean-object.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Removes `null`, `undefined` and empty entries from a given object.
+ *
+ * @param {object} object - Object which may contain `null`, `undefined` or empty values.
+ *
+ * @returns {object} Object without `null`, `undefined` and empty values.
+ */
```

Comment:
please check this PR https://github.com/infinum/eightshift-frontend-libs/pull/415 and add usage and output comments here